lib/wda/webdriveragent.js

```diff
@@ -240,6 +250,23 @@ class WebDriverAgent {
     return await this.xcodebuild.start();
   }
 
+  async prepareWDA () {
+    const wdaBundlePath = await fs.walkDir(this.derivedDataPath, true, (item) => item.endsWith('WebDriverAgentRunner-Runner.app'));
+
+    if (!wdaBundlePath) {
+      throw new Error(`Couldn't find the WDA bundle in the ${this.derivedDataPath}`);
+    }
+
+    if (!await this.device.isAppInstalled(WDA_BUNDLE_ID)) {
+      await this.device.installApp(wdaBundlePath);
+    }
+    await this.idb.installXCTestBundle(path.join(wdaBundlePath, 'PlugIns', 'WebDriverAgentRunner.xctest'));
```

Comment:
should we fallback to xcbuild if idb fails?

Reply:
Since I will put this behind a capability, we don’t have to. It is better if we fail fast